### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by NidhiShah0511**
Adding a README file with the project functionalities.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+### 7. Block a User’s Tuits
+As an administrator, I want to be able to block a given Tuit from a given user so that I can prevent the spread of violent and/or harmful content.
+
+<img width="908" alt="blockTuit" src="https://user-images.githubusercontent.com/12969516/206976118-b3f9293b-ca2a-4844-a1ae-f6514891fcb5.png">
+
+### Deployment
+
+To run this project on your machine, clone the main branch on your machine.
+1. Create a .env file containing the following variables - ENV, SECRET, DB_USERNAME, DB_PASSWORD.
```
Comment (cesarguerrero1) [inline README.md]: You also need to include an **_ORIGIN_URL_** environmental variable so that the CORS behaves.

**Review by cesarguerrero1 — APPROVED** [review]
If you can make that change from my comment, it looks good!

File: README.md (hunk 1 above)
Comment (NidhiShah0511) [reply README.md]: Added! Thanks

File: README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+As an administrator, I want to be able to retrieve a specific user so that I can perform a given task (Update a User, Delete a User, Etc.)
+
+<img width="873" alt="findAllUsers" src="https://user-images.githubusercontent.com/12969516/206975971-ddbd6d87-f59d-4b87-9c4e-2f6c559067dd.png">
+
+
+### 3. Update a User
+As an administrator, I want to be able to update a user so that I can alter the details associated with their account (Personal Information, Login, Password, Tuits (Content, Flagged Status, Blocked Status), Blocked Status)
+
+<img width="894" alt="updateUser" src="https://user-images.githubusercontent.com/12969516/206976007-061dfeee-9420-4713-b707-8ff3f8d0da96.png">
+
```
Comment (SpatikaH) [inline README.md:35]: Nitpick! Admin would be able to delete a user as well as a part of their CRUD operations! Loved the highlight in datamodel diagram :)

**Review by SpatikaH — APPROVED** [review]
Left a small comment, all good otherwise!

**Pull request merged**